### DISCUSSION TITLE
Support for .NET 6 and 7?

### DISCUSSION THREAD

**Discussion by ES0T** [dotnet/MQTTnet#1726] [General]
I just wanted to clarify this project supports and will work on .NET 6? What about .NET 7?

Personally, I find this line on the README a bit confusing :

`It is compatible with mostly any supported .NET Framework version `

When I see ".NET Framework", I think .NET Framework 4.x, and not .NET Core and/or .NET 5/6/7.

It might help others if this was rewritten to specify what versions of .NET are supported.

Thanks!

**Comment by chkr1011** [comment]
The library supports .NET 4.5.2 and never versions up to .NET 7.0.
